subscriptions/management/commands/add_missed_call_service_audio_notification_to_active_subscriptions.py

```diff
@@ -0,0 +1,32 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("Active subscription holders need to be informed via audio file "
+            "about the new missed call service.")
+
+    def handle(self, *args, **options):
+        self.stdout.write("Processing active subscriptions ...")
+        count = 0
+        try:
+            active_subscriptions_list = list(
+                Subscription.objects.filter(active=True))
```

Comment:
We should also filter where the message set is `audio`, subscriptions with text messagesets will get the bulk sms.

Reply:
I cannot seem to find any messageset of type audio in the SMB admins site. Do we create one? If so, what have we been using?